app/models/vacation.rb

```diff
@@ -25,6 +26,7 @@ class Vacation < ApplicationRecord
     allow_nil: true,
     if: :not_cancelled?
   validate :validate_start_date_close_to_weekend, if: :start_date
+  validate :validate_start_date_close_to_holiday, if: :start_date
```

Comment:
maybe we can consolidate both of these validations into one, we could call it `validate :start_date_close_to_off_day`

Reply:
also, I think it would be nice if we moved all this logic into a separate validator, it would keep this model a bit cleaner :eyes: